databases/week3/mealSharingDB.sql

```diff
@@ -0,0 +1,39 @@
+CREATE DATABASE IF NOT EXISTS `Meal_sharing`;
+
+CREATE TABLE
```

Comment:
Great job here Haile you are really good at creating tables and connecting them together